lib/src/__tests__/createClientRender.tsx

```diff
@@ -59,24 +59,7 @@ function clientRender(
     baseElement,
     queries: { ...queries, ...customQueries },
     wrapper: Wrapper,
-  }) as any;
-
-  /**
-   * convenience helper. Better than repeating all props.
-   */
-  result.setProps = function setProps(props: any) {
```

Comment:
Any reason for removing these helpers? You can look at the mono-repository codebase to see when it can be useful. We try to avoid it but I believe it's not always possible to avoid.

Reply:
It is about keep typescript typings. It is not possible to add something to the result, because it is `readonly` just quick feed